src/lib.rs

```diff
@@ -324,6 +324,43 @@ pub struct WalletOpts {
     #[cfg(feature = "esplora")]
     #[structopt(flatten)]
     pub esplora_opts: EsploraOpts,
+    #[cfg(feature = "compact_filters")]
+    #[structopt(flatten)]
+    pub compactfilter_opts: CompactFilterOpts,
+}
+
+/// Compact Filter options
+///
+/// Compact filter peer information used by [`OnlineWalletSubCommand`]s.
+#[cfg(feature = "compact_filters")]
+#[derive(Debug, StructOpt, Clone, PartialEq)]
+pub struct CompactFilterOpts {
+    /// Sets the SOCKS5 proxy for the Electrum client
+    #[structopt(name = "CPROXY_SERVER:PORT", long = "cmpct_proxy")]
+    pub proxy: Option<String>,
+
+    #[structopt(name = "PROXY_SERVER:USER", short = "u", long = "user")]
```

Comment:
Need to add docs so they show up in the --help output, something like:
If --cmpct_proxy is set, sets the SOCKS5 proxy user name